### PR TITLE
Update the description of optuna-dashboard on README.

### PR DESCRIPTION
**Pull request by c-bata**
## Motivation

Optuna Dashboard is no longer experimental.

## Description of the changes

Update the description of optuna-dashboard on README like the following:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -103,15 +103,14 @@ Examples can be found in [optuna/optuna-examples](https://github.com/optuna/optu
...
-It is still experimental, but much better in many regards.
+[Optuna Dashboard](https://github.com/optuna/optuna-dashboard) is a real-time web dashboard for Optuna.
+You can check the optimization history, hyperparameter importances, etc. in graphs and tables.
+You don't need to create a Python script to call [Optuna's visualization](https://optuna.readthedocs.io/en/stable/reference/visualization/index.html) functions.
 Feature requests and bug reports welcome!
 
-| Manage studies | Visualize with interactive graphs |
-| -------------- | --------------------------------- |
-| ![manage-studies](https://user-images.githubusercontent.com/5564044/97099702-4107be80-16cf-11eb-9d97-f5ceec98ce52.gif) | ![optuna-realtime-graph](https://user-images.githubusercontent.com/5564044/97099797-66e19300-16d0-11eb-826c-6977e3941fb0.gif) |
+![optuna-dashboard](https://user-images.githubusercontent.com/5564044/204975098-95c2cb8c-0fb5-4388-abc4-da32f56cb4e5.gif)
```
Comment (c-bata) [inline README.md:113]: optuna-dashboard

**Comment by HideakiImamura** [comment]
@cross32768 @gen740 Could you review this PR?

File: README.md [hunk 2]
```diff
@@ -103,15 +103,14 @@ Examples can be found in [optuna/optuna-examples](https://github.com/optuna/optu
...
 
-## Web Dashboard (experimental)
+## Web Dashboard
 
-The new Web dashboard is under the development at [optuna-dashboard](https://github.com/optuna/optuna-dashboard).
-It is still experimental, but much better in many regards.
+[Optuna Dashboard](https://github.com/optuna/optuna-dashboard) is a real-time web dashboard for Optuna.
+You can check the optimization history, hyperparameter importances, etc. in graphs and tables.
+You don't need to create a Python script to call [Optuna's visualization](https://optuna.readthedocs.io/en/stable/reference/visualization/index.html) functions.
 Feature requests and bug reports welcome!
```
Comment (gen740) [inline README.md:111]:
```suggestion
Feature requests and bug reports are welcome!
```
There is a slight grammar error.

**Review by gen740 — APPROVED** [review]
LGTM!

File: README.md (hunk 2 above)
Comment (gen740) [inline README.md:111]:
```suggestion
Feature requests and bug reports are welcome!
```
I guess there is a slight grammar error.

Comment (c-bata) [reply README.md:111]: I'm not sure if it's grammatically correct, but Crissman-san, a native English speaker, actually removed "are".
https://github.com/optuna/optuna/pull/2224#discussion_r559918136

Comment (gen740) [reply README.md:111]: Oh..., I see.

**Review by cross32768 — APPROVED** [review]
LGTM!

**Comment by c-bata** [comment]
Let me merge this PR since this PR has two approvals.

**Comment by c-bata** [comment]
Ah, I cannot approve my own pull request. @HideakiImamura Could you merge this PR?

**Review by HideakiImamura — APPROVED** [review]

**Pull request merged**